samples/AspNetCore3xODataSample.Web/Program.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace AspNetCore3xODataSample.Web
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.ConfigureKestrel(options =>
+                    {
+                        options.AllowSynchronousIO = true;
+                    });
```

Comment:
This should go closer to where you do I/O. Don't expect users to understand this detail or how to work around it.

Reply:
@saxu@microsoft.com - is this line still needed? We do now set AllowSynchronousIO in the Input and Output formatters.

---
In reply to: [348247017](https://github.com/OData/WebApi/pull/1963#discussion_r348247017) [](ancestors = 348247017)

Reply:
Yes. I will remove that later.

---
In reply to: [356331132](https://github.com/OData/WebApi/pull/1963#discussion_r356331132) [](ancestors = 356331132,348247017)